velox/exec/tests/CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ target_link_libraries(
   velox_functions_lib
   velox_functions_prestosql
   velox_hive_connector
+  velox_dwio_parquet_reader
```

Comment:
Should we add `if(VELOX_ENABLE_PARQUET)` check? If it's not turned on, the binary won't be built